addons/mail/static/src/utils/common/format.js

```diff
@@ -245,3 +245,19 @@ export function convertBrToLineBreak(str) {
 export function cleanTerm(term) {
     return unaccent((typeof term === "string" ? term : "").toLowerCase());
 }
+
+export function serverDateToLocalDateShortFormat(dateStr) {
```

Comment:
Meh ... seems like adding complexity and tools method ... can't we just use the short or medium format everytime and basta ?